packages/protobuf/src/messages-schema.ts

```diff
@@ -6,6 +6,7 @@ export enum DeviceModelInternal {
     T2B1 = 'T2B1',
     T3B1 = 'T3B1',
```

Comment:
I ran `yarn update-protobuf` locally to see if it is in sync.
- The addition of T3W1 is in sync (no diffs against _this PR_) :heavy_check_mark: 
- But it removed T3B1 :warning: 

:point_right: it's not related to this PR but someone should fix T3B1 in firmware repo